docs/insomnia/git-sync.md

```diff
@@ -10,6 +10,8 @@ category-url: get-started
 
 Sync with Git is a built-in feature for Design Documents that enables you to configure your repository to an external Git version control system like GitHub or BitBucket.
 
+On this page, you'll see reference to teams and team members. Learn more about [Team Collaboration](/insomnia/team-collaboration). Otherwise, you can use this feature as a solo developer to keep a copy of your work using Git versioning.
```

Comment:
Hmmm, "team members" in this page don't refer to the Team Collaboration feature 😢 maybe we can reword it. On this page, it just means anybody else that has access to the git repository.

Herein lies the irony between our two sync systems and how disconnected they are 😭

Reply:
Got it. I want to reserve "team" and "team members" for when we talk about our paid feature then. In other cases, we can use verbiage like "collaborator" or something else.